skyportal/handlers/api/spectrum.py

```diff
@@ -528,4 +539,4 @@ def get(self, obj_id):
 
             s["fluxes"] = s["fluxes"] / norm
 
-        return self.success(data=return_values)
+        return self.success(data={'obj_id': obj.id, 'spectra': return_values})
```

Comment:
Update docstrings to reflect new response structure?

Reply:
done